deployments/helm/proxy-nsmgr/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 apiVersion: v1
 appVersion: "1.0"
```

Comment:
I would change the appversion to 0.2.0 too

Reply:
Doh... I did that in the other PR for release-0.2

Mind if I do that in a follow on PR?  We'll need to update the skydive container as well... so there's a follow on PR coming :)